basic_hashtable/b_hashtables.py

```diff
@@ -32,16 +39,45 @@ def hash(string, max):
 # If you are overwriting a value with a different key, print a warning.
 # '''
 def hash_table_insert(hash_table, key, value):
-    pass
-
+    key_hash = hash(key, hash_table.capacity)
+    new_pair = Pair(key, value)
+    if hash_table.storage[key_hash]:
+        print("hash already exists")
```

Comment:
I believe the spec wants you to overwrite the existing value (in addition to the warning)